html/about.html

```diff
@@ -35,7 +40,7 @@ <h5 class="creative-logo-title">Creative Design & Content 2018 Logo</h5>
       <p class="creative-description">The logo of Design was decided through the logo competition</p>
```

Comment:
-  [ ] The logo should have a displayed border similar to the given design

Reply:
i do it too  right now thanks